components/badge/badge.scss

```diff
@@ -54,7 +54,7 @@
         align-items: center;
         line-height: 0;
         border-radius: rem(10);
```

Comment:
Not sure if it's a typo or scss way of declaring 10rem. Please have a look at this.